jupyterlab_gcedetails/src/components/auth_token_retrieval.tsx

```diff
@@ -1,6 +1,7 @@
 export function authTokenRetrieval(): Promise<string> {
   // eslint-disable-next-line prefer-const
   let authPopup: any;
```

Comment:
Can we add the proper types to these definitions. I believe `authPopup` will be a Window type and `timer` should be a number. You can look at what VSCode shows as the inferred type.